docker/start.sh

```diff
@@ -5,10 +5,29 @@ if [ -z "$REINDEX" ]; then
 	REINDEX=10
 fi
 
+if [ "${OPENGROK_WEBAPP_CONTEXT}" = "/" ]; then
+	WAR_NAME="ROOT.war"
+else
+	WAR_NAME="${OPENGROK_WEBAPP_CONTEXT#/}.war"
+fi
+
+if [ ! -f "/usr/local/tomcat/webapps/${WAR_NAME}" ]; then
+	date +"%F %T Deployment path changed. Redeploying..."
+
+	# Delete old deployment and (re)deploy
+	rm -rf /usr/local/tomcat/webapps/*
+	opengrok-deploy -c /opengrok/etc/configuration.xml \
+            /opengrok/lib/source.war /usr/local/tomcat/webapps/${WAR_NAME}
```

Comment:
Nit: perhaps quote the variable expansion?

Reply:
Of course. Not sure how I missed that. I'll fix it with a later patch set.